Xamarin.Forms.Platform.Android/Renderers/StreamImagesourceHandler.cs

```diff
@@ -13,7 +13,7 @@ public sealed class StreamImagesourceHandler : IImageSourceHandler
 			var streamsource = imagesource as StreamImageSource;
 			if (streamsource != null && streamsource.Stream != null)
 			{
-				using (Stream stream = await streamsource.GetStreamAsync(cancelationToken).ConfigureAwait(false))
+				using (Stream stream = await ((IStreamImageSource)streamsource).GetStreamAsync(cancelationToken).ConfigureAwait(false))
```

Comment:
Not right here; streamSource is a local variable. I've gone through the other PRs and added properties where they made sense.